docs/_static/credential/v1.yaml

```diff
@@ -0,0 +1,246 @@
+openapi: "3.0.0"
+info:
+  title: Nuts Verifiable Credential API spec
+  description: API specification for Verifiable Credentials
+  version: 1.0.0
+  license:
+    name: GPLv3
+paths:
```

Comment:
I think we're missing a call for verifying VCs?

Or do we only return verified VCs? If so maybe we should add a verificationDate to the VerificationOutcome and return it with the VC.

Reply:
I'm not sure if we need it. We'll see that is required when we integrate the node.